_services/custom-software-development.md

```diff
@@ -1,26 +1,31 @@
 slug: custom-software-development
 related-projects:
   essayjack: '/essayjack'
 ---
 
-## About
-The path of the righteous man is beset on all sides by the iniquities of the selfish and the tyranny of evil men. Blessed is he who, in the name of charity and good will, shepherds the weak through the valley of darkness, for he is truly his brother's keeper and the finder of lost children. And I will strike down upon thee with great vengeance and furious anger those who would attempt to poison and destroy My brothers. And you will know My name is the Lord when I lay My vengeance upon thee.
+# Custom Software Development
 
-The path of the righteous man is beset on all sides by the iniquities of the selfish and the tyranny of evil men. Blessed is he who, in the name of charity and good will, shepherds the weak through the valley of darkness, for he is truly his brother's keeper and the finder of lost children. And I will strike down upon thee with great vengeance and furious anger those who would attempt to poison and destroy My brothers. And you will know My name is the Lord when I lay My vengeance upon thee.
+Whether for a new initiative or to help with an existing project, when we join your team our aim is to bring your business forward. We are value oriented and wish to become not just a deliverer of tasks, but a thought partner in your company’s mission.
 
+Some of the values we bring with us to every situation we encounter are:
 
-## Process
-The path of the righteous man is beset on all sides by the iniquities of the selfish and the tyranny of evil men. Blessed is he who, in the name of charity and good will, shepherds the weak through the valley of darkness, for he is truly his brother's keeper and the finder of lost children. And I will strike down upon thee with great vengeance and furious anger those who would attempt to poison and destroy My brothers. And you will know My name is the Lord when I lay My vengeance upon thee.
+* [Async communication](https://doist.com/blog/asynchronous-communication/). We understand that good communication rules above all else. We take the time to communicate fully, accurately and clearly, to allow other team members to digest information on their own schedule.
+* Creative problem solving. We think creatively about problems, in order to achieve desired results with less effort and in a shorter time frame.
+* Trustworthiness and dependability. We strive to do the Right Thing in every situation. We avoid cutting corners that will hurt us later on. We are clear about the trade-offs and consequences in the decisions we make. We take your concerns and interests to heart. We sometimes break our own rules if the situation warrants.
+* [Continuous delivery](https://continuousdelivery.com/). We lean on sound engineering practices to get changes into production safely and quickly. Techniques like feature toggles, test coverage, similar environments, comprehensive business metric monitoring to name a few.
```

Comment:
This may probably be just me being pedantic, but I feel instead of saying `to name a few`, we just name them.

Reply:
It may be more a stylistic thing rather than being pedantic. The sentence makes sense without it, and by removing it becomes shorter and therefore easier to read. I'll remove it...